README.md

```diff
@@ -72,6 +72,34 @@ session
   .catch(function(error) {
     console.log(error);
   });
+
+  //run queries in a transaction
+  var tx = session.beginTransaction();
+  tx.run("CREATE (alice {name : {nameParam} })", { nameParam:'Alice'}");
+  tx.run("MATCH (alice {name : {nameParam} }) RETURN alice.age", { nameParam:'Alice'}");
+  //decide if the transaction should be committed or rolled back
+  var success = ...
+  ...
+  if (success) {
+    tx.commit()
+      .subscribe({
+        onNext: function(record) {
+          console.log(record);
```

Comment:
This should not get called for the commit right? Eg. there's no records in the result from a commit.